### PR TITLE
Healthcare Toolkit ReadMe

### PR DESCRIPTION
**Pull request by svetakvsundhar**
Fixes https://github.com/GoogleCloudPlatform/healthcare/issues/573

Rendering snippet below.

Screenshot 2024-03-26 at 3 54 54 PM

### CODE REVIEW

File: utilities/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The healthcare toolkit is designed to guide various data movement patterns including data ingestion, consumption, and auditing. The utilities provide data movement patterns between various sources/sinks and Google Cloud healthcare products. 
+Healthcare Toolkit Utilities.
+
+## Ingest
+1. [Kafka to FHIR Store](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/ingest/dataflow_kafka_to_fhirstore) - Consume messages via Kafka, and write to a GCP FHIR Store.
+2. [Pub/Sub to FHIR Store](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/ingest/dataflow_pubsub_to_fhirstore) - Consume messages via Pub/Sub, and write to a GCP FHIR Store.
+
+## Consume
+1. [FHIR Store to GCS](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/consume/dataflow_fhirstore_bq_consume) - Consumes a notification from a GCP FHIR Store, and writes the message to GCS.
```
Comment (devanshmodi) [inline utilities/README.md:11]: Consumes a notification containing FHIR resource in JSON format along with attributes from Google Cloud Healthcare FHIR Store, reads the latest messages from BigQuery, transforms it and writes the message to GCS.

Comment (devanshmodi) [reply utilities/README.md:11]: @surjits254 Feel free to add your comments this utility.

File: utilities/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+2. [FHIR Store to Pub/Sub + GCS](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/consume/dataflow_fhirstore_to_gcs_pubsub) - Consumes a notification from a GCP FHIR Store, and writes it to GCP Pub/Sub and Google Cloud Storage.
```
Comment (devanshmodi) [inline utilities/README.md:12]: Consumes a notification containing FHIR resource in JSON format along with attributes from Google Cloud Healthcare FHIR Store, transforms it and writes the message to Google Cloud Storage and Google Cloud Pub/Sub.

**Review by devanshmodi — COMMENTED** [review]
Have reviewed and added comments to rephrase some of the descriptions.

File: utilities/README.md (hunk 1 above)
Comment (surjits254) [reply utilities/README.md:11]: lgtm, what you are suggesting

Comment (devanshmodi) [reply utilities/README.md:11]: @surjits254 nothing as you have worked on this one, tagged you incase you want to add more details.

Comment (surjits254) [reply utilities/README.md:11]: Yes, I am good with changes you are suggesting!!

**Comment by svetakvsundhar** [comment]
Thanks, PTAL

**Review by surjits254 — APPROVED** [review]
LGTM

**Pull request merged**